en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,229 @@
+subjected to financial penalties.
+
+WSO2 Identity Server (WSO2 IS) is designed based on privacy best practices, and is fully compliant with GDPR. GDPR compliance in IAM and API security spaces can be completely fulfilled with WSO2 IS. This section outlines the major GDPR-related features that WSO2 IS supports, and also provides necessary references to specific concepts within the section.
+
+!!! Info
+    To understand GDPR more elaborately, take a look at this tutorial series on *Creating a Winning GDPR Strategy*.
+
+    -   Part 1 - [Introduction to
+        GDPR](https://wso2.com/library/article/2017/12/introduction-to-gdpr/)
+
+    -   Part 2 - [7 Steps for GDPR
+    Compliance](https://wso2.com/library/article/2017/12/7-steps-for-gdpr-compliance/)
+
+    -   Part 3 - [Identity and Access Management to the
+        Rescue](https://wso2.com/library/article/2018/2/identity-and-access-management-to-the-rescue/)
+
+    -   Part 4 - [GDPR Compliant Consent
+        Design](https://wso2.com/library/articles/2018/03/creating-a-winning-gdpr-strategypart-4-gdpr-compliant-consent-design/)
+
+    For more resources on GDPR, see the white papers, case studies, solution
+    briefs, webinars, and talks published on our [WSO2 GDPR homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/). See the original GDPR legal text
+    [here](http://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32016R0679).
+
+## Privacy by design and privacy by default
+
+As a leading open source Identity and Access Management (IAM) product,
+WSO2 IS has been designed and architectured based on well
+known *Secure by Design* and *Privacy by Design* principles.  With the
+formalization of GDPR in 2016, the product architecture of WSO2 Identity
+Server has been reviewed and fine-tuned accordingly to support
+privacy principles in an efficient manner, with less overhead to product
+performance and user experience. WSO2 IS provides all privacy features
+enabled in the product as default options, and uses up-to-date
+algorithms and frameworks for all cryptographic operations such as data
+encryption, signing, and hashing.
+
+## Consent identity management
+
+Generally, identity data are scattered over several systems within an
+organization. In order to reach GDPR compliance, it is required to
+review, redesign, and modify each of these systems. This is a
+maintenance overhead and consumes a significant portion of annual IT
+budget and requires a specialized set of skills for continuous review
+and modification process.
+
+Moving to a GDPR-compliant IAM solution, so that all identity profiles
+are managed centrally and share only required data with other systems in
+an on-demand manner through well-known security standards such as SAML and 
+OpenIDConnect greatly reduces the maintenance overhead discussed above and
```

Comment:
OpenIDConnect --> OpenID Connect

Reply:
IMO, the main reason for consent management is not to reduce the maintenance overhead right? Not entirely clear on how the first para connects to describing consent identity management